manifests/init.pp

```diff
@@ -44,6 +44,8 @@
   $gid          = undef,
   $shell        = '/bin/true',
 
+  $no_websudo   = false,
```

Comment:
Mirroring the the ugliness of the original variable is probably the better option: eg. ```$jira_websudo_is_disabled```